main.go

```diff
@@ -123,7 +127,15 @@ func main() {
 		apiroots[i] = path.Join(root, appname, apiversion)
 	}
 
-	srv, err := NewServer(addr, apiroots, db)
+	msgChannel := make(chan []byte, 100)
+	producer := &ProducerConfig{
+		Brokers: []string{kafkaBootstrap},
+		Topic:   metricsTopic,
+		Async:   true,
+	}
+	go Producer(producer, msgChannel)
```

Comment:
Wrap this in a conditional `if` block.

```go
if kafkaBootstrap != "" {
    c := make(chan []byte, 100)
    go Producer(&ProducerConfig{[]string{kafkaBootstrap}, metricsTopic, true}, c)
}
```

Reply:
I had this done this way earlier and backed away from it for some reason. I definitely agree